pkg/cri/sbserver/helpers.go

```diff
@@ -95,6 +95,11 @@ const (
 
 	// runtimeRunhcsV1 is the runtime type for runhcs.
 	runtimeRunhcsV1 = "io.containerd.runhcs.v1"
+
+	// name prefix for CRI sbserver specific spans
+	criSbServerSpanPrefix = "pkg.cri.sbserver"
```

Comment:
I wanted to follow a common naming convention for the spans, so used : "relative path to the package"+ "name describing the span logic or the method", so added the name. Any reason we do not want to expose server vs sbserver difference? I was assuming this could help catch divergence issues between the both.

Reply:
Because we eventually migrate from pkg.cri.server to pkg.cri.sbserver and don't want to surprise the consumer of the traces.

That being said, I do agree having different names will help users spot the difference in between.